securedrop_client/logic.py

```diff
@@ -497,13 +515,15 @@ def on_message_download_failure(self, exception: Exception) -> None:
         """
         logger.debug('Failed to download message: {}'.format(exception))
 
+        # Keep resubmitting the job if the download is corrupted.
```

Comment:
Just want to note that in the failure case, the job will retry 5 times and then will be added again to the end of the queue allowing other jobs to get processed still. This could continue indefinitely but other jobs will continue to be processed.